### DISCUSSION TITLE
Unable to view live data in Chrome on Android

### DISCUSSION THREAD

**Discussion by craigblaster** [foxglove/community#964] [Q&A]
**Description**
Connecting to a live ROS2 data source from my Android phone running Chrome gives me a prompt to enable "unsafe scripts" using the shield icon. Screenshot attached (in comment).
There is no shield icon or similar setting that I could find.

- Version: web
- Platform: Android / Chrome 122
- Data source (e.g. bag file, mcap file, rosbridge, ROS 1/2 native): ROS2 native

**Steps To Reproduce**
1. Run the foxglove bridge on a ROS2 machine.
2. Using the webapp app.foxglove.dev, choose 'open connection' and enter the URL/IP
3. Error message is shown

**Expected Behavior**
* Show the correct instructions to enable the live data view

**Comment by linear** [comment]
<p><a href="https://linear.app/foxglove/issue/FG-7046/unable-to-view-live-data-in-chrome-on-android">FG-7046 Unable to view live data in Chrome on Android</a></p>

**Comment by craigblaster** [comment]
![Screenshot_20240306-214555_Chrome](https://github.com/foxglove/studio/assets/3603651/937f9b9e-7e11-4b7c-b0a6-e61336877a64)

**Comment by jtbandes** [comment]
Are you able to find relevant settings by following these instructions? https://support.google.com/chrome/answer/114662?hl=en&co=GENIE.Platform%3DAndroid

**Comment by craigblaster** [comment]
None of the settings or permissions there seem related to the problem.
I have set everything to Allowed or Ask but the problem persists.

**Comment by chfritz** [comment]
One thing you could try is to open about:flags in your browser and add your ws:// url to the list of "Insecure origins treated as secure".

**Reply by craigblaster** [reply]
Thanks, tried it but no joy.

**Reply by outrider-jkoch** [reply]
This worked for me but I had to include the port as well (e.g. ws://computer:8765) and make sure the setting is enabled and then relaunch.

**Comment by RossThayer** [comment]
@craigblaster have you had any luck getting this to work? I am running into the same issue.

**Reply by craigblaster** [reply]
Unfortunately not

**Comment by Xin-jiasong** [comment]
我首先开启了`chrome://flags`中的`Insecure origins treated as secure`
然后在下面的菜单中分别加入了`https://foxglove.dev`以及`ws://computer:8765`
就可以加载话题了
